docs/content/configuration/inheritance/_index.md

```diff
@@ -0,0 +1,425 @@
+---
+title: "Inheritance"
+date: 2022-05-02T20:00:00+02:00
+weight: 16
+---
+
+{{% notice style="tip" %}}
+You can use `resticprofile [<profile-name>.]show` to see the effect inheritance on a profile
+{{% /notice %}}
+
+## Profile Inheritance
+
+Profiles can inherit from a parent profile. This allows to define the general behavior and common configuration in a base profile while **derived** profiles only define what is specific, e.g. what needs to be included in the backup or which command [hooks](../hooks/) (e.g. `run-before`, `run-after` & `run-finally`) must be started.
```

Comment:
you can use the macro {{< ref "/configuration/hooks" >}} to generate a url. The good thing is that it's going to fail when we rename a chapter 😉

Reply:
Was using IDE auto-complete but yeah, compile time check makes sense 😀